app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    # @users = User.all
+  end
+
+  def show
+    # @user = User.find(params[:id])
+  end
```

Comment:
- [OPTIONAL] Please kindly remove commented code from your code base as this makes your code base dirty and less readable.